core-bundle/src/Resources/config/listener.yml

```diff
@@ -70,10 +70,21 @@ services:
 
     contao.listener.exception_converter:
         class: Contao\CoreBundle\EventListener\ExceptionConverterListener
+        arguments:
+            - "@database_connection"
```

Comment:
I assume you have check that we are not ending in a loop if the database class throws an exception while we are converting the exception?

Reply:
I have not, but I'm pretty sure the Symfony exception handler handles this case. But feel free to verify.